crates/forge_infra/src/env.rs

```diff
@@ -82,6 +82,9 @@ impl ForgeEnvironmentInfra {
             forge_api_url,
             custom_history_path,
             max_conversations: parse_env::<usize>("FORGE_MAX_CONVERSATIONS").unwrap_or(100),
+            title_format: parse_env::<String>("FORGE_TITLE_FORMAT").unwrap_or_else(|| {
+                "[{timestamp} {input}/{output} {cost} {cache_pct}] {title} {subtitle}".to_string()
+            }),
```

Comment:
Proposal:

1. Create a new template engine using mustache, which supports colors.
2. Create a new `.hbs` or `.md` file with the template that looks kind of like 

    ```hbs
    dim([{timestamp} {input}/{output} {cost} {cache_pct}]) {title} {subtitle}
    ```
    Perhaps a more sophisticated template that also handles levels
    ```hbs
    {{if level == "info"}}
        white([{timestamp} {input}/{output} {cost} {cache_pct}]) {title} {subtitle}
    {{else if level == "warning"}}
        yellow([{timestamp} {input}/{output} {cost} {cache_pct}]) {title} {subtitle}
    {{/if}}
    ```